<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JacqQVisDialogBase</class>
 <widget class="QDialog" name="JacqQVisDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Jacquez's Q Visualization</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>771</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QSlider" name="DateSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateSelector">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TagGlobalStat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Global Q Statistic:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TagDateStat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Slice Qt Statistic:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ValueDateStat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>None Selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>No Mesages</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="GlobalResultsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>771</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>JacqQVisDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>JacqQVisDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
